tests/class/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -341,6 +349,15 @@ abstract class ReallyReallyReallyLongClassName
     {
         return new SplQueue();
     }
+
+    public function longLongAnotherFunction(
+        string $foo,
+        string $bar,
+        int $baz
+    ): string
+    {
```

Comment:
This is acutally a bug, isn't it?

Reply:
@czosel yep, we have todo around this, let's do this in other PR?

Reply:
Alright! Can you open an issue for this? (Or does one exist already?)

Reply:
@czosel Done https://github.com/prettier/plugin-php/issues/171 :+1:

Reply:
Thanks :)